source/fundamentals/auth.txt

```diff
@@ -32,8 +32,10 @@ Edition are as follows:
 * :ref:`MONGODB-AWS <mongodb-aws-auth-mechanism>`
 * :ref:`X.509 <x509-auth-mechanism>`
 
-To authenticate using ``Kerberos`` or ``LDAP``, see the
-:doc:`Enterprise Authentication Mechanisms guide </fundamentals/enterprise-auth>`.
+.. TODO (DOCSP-29270) Add back when Enterprise Auth is available
+.. To authenticate using ``Kerberos`` or ``LDAP``, see the
+.. :doc:`Enterprise Authentication Mechanisms guide </fundamentals/enterprise-auth>`.
+
 For more information on establishing a connection to your MongoDB cluster,
 read our :doc:`Connection Guide </fundamentals/connection>`.
 
```

Comment:
I can't comment on it, but the Rst included in line 45 links to the Java api documentation instead of the kotlin coroutine docs

Reply:
Good catch. Updated